package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@nucypher/nucypher-contracts",
-  "version": "0.16.0",
+  "version": "0.20.0",
```

Comment:
After running `npm install` and `npm test` the version was updated. I assume it just updates to the latest published version (cc @piotr-roslaniec ).

Reply:
Ok. 0.20.0 makes sense because I originally based the PR over tags/v0.20.0 because the latest contracts in `main` are not yet deployed anywhere. Both lynx and tapir use 0.20.0 and not what was on `main`. 

Once the 0.20.0 contracts were deployed for Lynx I then rebased the PR over `main`.